Directory.Build.props

```diff
@@ -1,7 +1,10 @@
 <Project>
   <!-- Directory.Build.props contains the common build settings for all projects in the repo. -->
 
-  <Import Project="build/BranchInfo.props" />
```

Comment:
I think all 3 of these files `BranchInfo`, `BranchInfo.Stable` and `StablePackagesInfo` could all be combined into a single file - `build/BranchInfo.props`.

Reply:
Oh... I interpreted this as being at the *nuget* level, not the assembly level. Is that wrong? SHouldn't it be at nuget level?

Reply:
It needs to be for both. We use these versions for both NuGet package versions and for AssemblyVersions.

So if you only did it for nuget - you would have a `1.0.0` NuGet package with an assembly inside of it with AssemblyVersion `0.13.0.0`.